pkg/apis/core/helper/helpers.go

```diff
@@ -113,34 +113,34 @@ var Semantic = conversion.EqualitiesOrDie(
 	},
 )
 
-var standardResourceQuotaScopes = sets.NewString(
-	string(core.ResourceQuotaScopeTerminating),
-	string(core.ResourceQuotaScopeNotTerminating),
-	string(core.ResourceQuotaScopeBestEffort),
-	string(core.ResourceQuotaScopeNotBestEffort),
-	string(core.ResourceQuotaScopePriorityClass),
+var standardResourceQuotaScopes = sets.New(
```

Comment:
For what it's worth now, it would have been nice to break this up into a series of commits that each do one thing or one package.

Reply:
It's hard to split this into `each do one thing` now. But I can definitely split to `each do one package/file` after comments are solved.

Reply:
Don't bother now - for future work, especially cleanups and refactorings, lots of small commits is better than one big one.